website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Concepts"
+linkTitle: "Concepts"
+weight: 35
+---
+
+By adding Karpenter to a Kubernetes cluster you can dramatically improve the efficiency and cost of running workloads on that cluster.
+Instead of introducing lots of new concepts and configuration, Karpenter is tightly integrated with Kubernetes features to make sure that the right types and amounts of compute resources are available to pods as they are needed.
```

Comment:
Consider dropping `Instead of introducing lots of new concepts and configuration`. I think it's as strong without, and it tends to be more crisp to say what something is, rather than what it isn't.